src/main/java/uk/gov/hmcts/probate/services/pin/controllers/PinController.java

```diff
@@ -11,15 +16,37 @@
 public class PinController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PinController.class);
+    
     @Autowired
     private PinService pinService;
 
-    @RequestMapping(path = "/pin/{phoneNumber}", method = RequestMethod.GET)
-    public String invite(@PathVariable String phoneNumber,
-                         @RequestHeader("Session-Id") String sessionId) throws NotificationClientException {
-
-        LOGGER.info("Processing session id " + sessionId);
+    @RequestMapping(path = "/pin", method = RequestMethod.GET)
+    public ResponseEntity<String> invite(@RequestParam String phoneNumber,
+                         @RequestHeader("Session-Id") String sessionId) throws NotificationClientException, UnsupportedEncodingException {
+ 	
+		if ( null == sessionId ) {
```

Comment:
Please switch these two variables around. sessionId should be compared to null and not the other way around (this is a readability issue more than anything).

Reply:
Also remove spaces here as well please.